modules/80_optimization/nash/output.gms

```diff
@@ -24,7 +24,7 @@ putclose prices_NASH;
 *** The file is formated in columns as follows: | Experiment title | Region | Year | Iteration | Market | surplus | price |  ..
 file nash_info_convergence / "nash_info_convergence.csv" / ;
 put nash_info_convergence;
-put 'Scenario',',','Region',',','Year',',','Iteration',',',"Market",",","p80_surplus",",","p80_pvp_itr",",","p80_surplusMax",",","p80_surplusMaxRel",",":0 ;
+put 'Scenario',',','Region',',','Year',',','Iteration',',',"Market",",","p80_surplus",",","p80_pvp_itr",",","p80_surplusMax_iter",",","p80_surplusMaxRel",",":0 ;
```

Comment:
This look painful.
https://www.gams.com/latest/docs/UG_Put.html#INDEX__2f_pc_22_suffixes_2d__21_put_2e_file

> 5: Formatted output; non-numeric output is quoted and each item is delimited with commas.

Reply:
but we are anyway looking at other parameter with the same dimensions